docs/users-guide/11-metabot.md

```diff
@@ -1,43 +1,45 @@
 ## Getting answers in Slack with MetaBot
 
-You can already send data to Slack on  a set schedule via [Pulses](10-pulses.md) but what about when you need an answer right now? Say hello to MetaBot.
+You can already send data to Slack on  a set schedule via [Pulses](10-pulses.md), but what about when you need an answer right now? Say hello to MetaBot.
 
 MetaBot helps add context to conversations you’re having in Slack by letting you insert results from Metabase.
 
 ### Connecting to Slack.
 To use MetaBot with Slack you’ll first need to connect Metabase to your Slack with an API token.
 
-See [Setting up Slack](http://www.metabase.com/docs/latest/administration-guide/07-setting-up-slack) for more information.
+See [Setting up Slack](http://www.metabase.com/docs/latest/administration-guide/09-setting-up-slack) for more information.
 
 
 ### What can MetaBot do?
-
 MetaBot can show individual questions and also lists of questions that have already been asked in Metabase.
 
 If you ever need help remembering what MetaBot can do, just type ```metabot help``` in Slack.
 
 ![MetaBot help](images/metabot/MetabotHelp.png)
 
+### Where can I use MetaBot?
+You can talk to MetaBot in any Slack channel, including private ones, as long as you've invited the MetaBot Slack user to that channel.
+
 ### Showing questions
 
 To see a question from Metabase in Slack type
-'''metabot show "<question-name>"''' where question name is the title of one of saved questions. If you have several similarly named questions Metabot will ask you to differentiate between the two by typing the number next to the name.
+```metabot show [question name]``` where ```question name``` is the title of one of your saved questions. If you have several similarly named questions, Metabot will ask you to differentiate between the two by typing the number next to the name.
 
 ![MetaBot similar](images/metabot/MetabotSimilarItems.png)
 
-That number is the ID number of the question in Metabase and if you find yourself using the same question over and over again you can save a bit of time by typing “MetaBot show 19.”
+That number is the ID number of the question in Metabase, and if you find yourself using the same question over and over again you can save a bit of time by typing “MetaBot show 19.”
 
 ![MetaBot show](images/metabot/MetabotShow.png)
 
 ## Listing questions
-If you don’t have a sense of which questions you want to view in  Slack, you can type ```MetaBot list``` to get a list of recent questions from your Metabase.
+If you don’t have a sense of which questions you want to view in  Slack, you can type ```MetaBot list``` to get a list of the most recently saved questions in your Metabase.
 
 ![MetaBot show](images/metabot/MetabotList.png)
 
 
 ## To review
 
-- [Connect to Slack](10-pulses.md) to start using MetaBot.
+- [Connect to Slack](http://www.metabase.com/docs/latest/administration-guide/09-setting-up-slack) to start using MetaBot.
```

Comment:
I'm fairly sure if we link to /latest here that'll break version specific docs, but I don't know if that's 100% true. @salsakran can you shed some light on that?